test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -4431,6 +4431,70 @@ public async Task Purge_Single_Instance_History(bool extendedSessions, string st
             }
         }
 
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [MemberData(nameof(TestDataGenerator.GetBooleanAndFullFeaturedStorageProviderOptions), MemberType = typeof(TestDataGenerator))]
+        public async Task Purgemultipleinstancehistory(bool extendedSessions, string storageProvider)

```

Comment:
Please follow C# method naming guidelines, which are to use PascalCase. This will make the code more readable to other developers.
```suggestion
        public async Task PurgeMultipleInstanceHistories(bool extendedSessions, string storageProvider)

```